### ISSUE TITLE
Can't sign up to Fig with email or github account

### ISSUE THREAD

**Issue by angieborolyuk57** [withfig/fig#2713]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

MacOS

### Expected behaviour

User should be able to sign up with the email or github account 

### Actual behaviour

There is an error while trying to sign up. A message is logged: we do not accept sign ups currently or something like this. While trying to sign up with github, there is a an error with white screen. impossible to launch fig dashboard 
<img width="1026" alt="Screenshot 2023-09-02 at 21 07 14" src="https://github.com/withfig/fig/assets/116447706/a3093786-db6d-4397-8607-0709dd38c02e">

**Comment by Adam-Madet** [comment]
Same

**Comment by mschrage** [comment]
[Fig is joining AWS](https://fig.io/blog/post/fig-joins-aws) - we're working with internal security teams to re-enable login. Thanks for your patience and stay tuned.

**Comment by digracesion** [comment]
Yep, encountering this issue as well on both the web and the downloaded app.

**Comment by Leoyang183** [comment]
Is it possible to register by using an email address?

**Comment by code1379** [comment]
3 month left, still can't use. the effecient as fast as Zootopia flash

**Comment by abhinav626** [comment]
check this out from AWS that integrates fig 

https://aws.amazon.com/blogs/devops/introducing-amazon-codewhisperer-for-command-line/
